test/functional/page_objects/common_page.js

```diff
@@ -37,6 +37,28 @@ export function CommonPageProvider({ getService, getPageObjects }) {
   const defaultFindTimeout = config.get('timeouts.find');
 
   class CommonPage {
+
+    static async navigateToUrlAndHandleAlert(url, shouldAcceptAlert) {
+      log.debug('Navigate to: ' + url);
+      try {
+        await browser.get(url);
+      } catch(e) {
+        log.debug('Error navigating to url');
+        try {
```

Comment:
Is there a way to test that an alert exists? Would be nice to know failure is because of alert and not something else.